netconfig.md

```diff
@@ -12,7 +12,7 @@ You need to manually permit the Spacemesh App to access the internet by opening
 1. Navigate to Control Panel, System and Security and Windows Firewall.
 2. Select Advanced settings and highlight Inbound Rules in the left pane.
 3. Right click Inbound Rules and select New Rule.
-4. Add the port you need to open and click Next.
+4. Add the port you need to open and click Next. 
```

Comment:
??
I see a windows saying "What type of rule would you like to create?" with 4 options, one is Port.

Reply:
you are right, we need to add it as a step - select 'port' as the rule type....

Reply:
Pull my edits that attempt to address this issue in this branch